test/index.js

```diff
@@ -0,0 +1,5 @@
+import {} from './spec/lateralus';
```

Comment:
Is there a functional difference between `{} from` and just `import './spec/lateralus';`?